### ISSUE TITLE
Gotenberg Docker Builds No Longer Work

### ISSUE THREAD

**Issue by sgrigson** [gotenberg/gotenberg#457]
Try:

docker run --rm -p 3000:3000 gotenberg/gotenberg:7

I've tried using old versions, no luck. Always get this after calling the libreoffice/convert endpoint:

`{"level":"error","ts":1652141460.9718838,"logger":"api","msg":"convert to PDF: unoconv PDF: unix process error: wait for unix process: exit status 5","trace":"93b2e2d8-b3d4-4767-b65e-2d03b468cd11","remote_ip":"172.17.0.1","host":"localhost:3000","uri":"/forms/libreoffice/convert","method":"POST","path":"/forms/libreoffice/convert","referer":"","user_agent":"PostmanRuntime/7.29.0","status":500,"latency":1453830593,"latency_human":"1.453830593s","bytes_in":341,"bytes_out":21}`

Note that this happens regardless of the actual arguments passed to the gotenberg process, far as I can tell.

The way it's behaving, I'm wondering if some underlying dependency changed in the core. Nothing I do seems to make any difference...if I'm running it as Docker, it does this.

**Comment by sgrigson** [comment]
Full dump with all debugging turned on:

```
pdf-exporter-svc  | 
pdf-exporter-svc  |   _____     __           __               
pdf-exporter-svc  |  / ___/__  / /____ ___  / /  ___ _______ _
pdf-exporter-svc  | / (_ / _ \/ __/ -_) _ \/ _ \/ -_) __/ _ '/
pdf-exporter-svc  | \___/\___/\__/\__/_//_/_.__/\__/_/  \_, / 
pdf-exporter-svc  |                                    /___/
pdf-exporter-svc  | 
pdf-exporter-svc  | A Docker-powered stateless API for PDF files.
pdf-exporter-svc  | Version: 7.5.2
pdf-exporter-svc  | -------------------------------------------------------
pdf-exporter-svc  | [SYSTEM] modules: api chromium gc libreoffice logging pdfcpu pdfengines pdftk prometheus qpdf uno uno-pdfengine webhook 
pdf-exporter-svc  | [SYSTEM] api: server listening on port 8081
pdf-exporter-svc  | [SYSTEM] uno: long-running LibreOffice listener disabled
pdf-exporter-svc  | [SYSTEM] pdfengines: pdfcpu pdftk qpdf uno-pdfengine
pdf-exporter-svc  | [SYSTEM] prometheus: collecting metrics
pdf-exporter-svc  | [SYSTEM] gc: application started
pdf-exporter-svc  | {"level":"debug","ts":1652142273.0859609,"logger":"api.formslibreofficeconvert","msg":"form data values: map[landscape:[false]]","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142273.0860078,"logger":"api.formslibreofficeconvert","msg":"form data files: map[file-sample_500kB.doc:/tmp/9fa28c36-c325-4737-8e32-421d438f1d4c/file-sample_500kB.doc]","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142273.0862224,"logger":"api.formslibreofficeconvert.usrliblibreofficeprogramsoffice.bin","msg":"start unix process: /usr/lib/libreoffice/program/soffice.bin --headless --invisible --nocrashreport --nodefault --nologo --nofirststartwizard --norestore -env:UserInstallation=file:///tmp/86850c57-60c7-455f-bb4c-44e4e9bbdcc9 --accept=socket,host=127.0.0.1,port=35843,tcpNoDelay=1;urp;StarOffice.ComponentContext","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142274.0467908,"logger":"api.formslibreofficeconvert.usrliblibreofficeprogramsoffice.bin","msg":"unix process already killed","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142274.0468335,"logger":"api.formslibreofficeconvert","msg":"got exit code 81, e.g., LibreOffice listener first start","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142274.0469472,"logger":"api.formslibreofficeconvert.usrliblibreofficeprogramsoffice.bin","msg":"start unix process: /usr/lib/libreoffice/program/soffice.bin --headless --invisible --nocrashreport --nodefault --nologo --nofirststartwizard --norestore -env:UserInstallation=file:///tmp/86850c57-60c7-455f-bb4c-44e4e9bbdcc9 --accept=socket,host=127.0.0.1,port=35843,tcpNoDelay=1;urp;StarOffice.ComponentContext","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142274.0473409,"logger":"api.formslibreofficeconvert","msg":"waiting for the LibreOffice listener socket to be available...","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.1795502,"logger":"api.formslibreofficeconvert","msg":"LibreOffice listener socket available","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.17962,"logger":"api.formslibreofficeconvert","msg":"print to PDF with: {Landscape:false PageRanges: PDFformat:}","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.1797023,"logger":"api.formslibreofficeconvert.usrbinunoconv","msg":"start unix process: /usr/bin/unoconv --no-launch --format pdf --port 35843 -vvv --output /tmp/9fa28c36-c325-4737-8e32-421d438f1d4c/38a42685-5cfc-43b4-8681-5ff274ceaaed.pdf /tmp/9fa28c36-c325-4737-8e32-421d438f1d4c/file-sample_500kB.doc","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.2299469,"logger":"api.formslibreofficeconvert.usrbinunoconv.stderr","msg":"Verbosity set to level 3","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.2304738,"logger":"api.formslibreofficeconvert.usrbinunoconv.stderr","msg":"DEBUG: Connection type: socket,host=127.0.0.1,port=35843,tcpNoDelay=1;urp;StarOffice.ComponentContext","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.2828298,"logger":"api.formslibreofficeconvert.usrbinunoconv.stderr","msg":"Input file: /tmp/9fa28c36-c325-4737-8e32-421d438f1d4c/file-sample_500kB.doc","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3713148,"logger":"api.formslibreofficeconvert.usrbinunoconv.stderr","msg":"unoconv: UnoException during import phase:","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3713512,"logger":"api.formslibreofficeconvert.usrbinunoconv.stderr","msg":"The document 'file:///tmp/9fa28c36-c325-4737-8e32-421d438f1d4c/file-sample_500kB.doc' could not be opened.","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3715231,"logger":"api.formslibreofficeconvert.usrbinunoconv.stdout","msg":"Using office base path: /usr/lib/libreoffice","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3715558,"logger":"api.formslibreofficeconvert.usrbinunoconv.stdout","msg":"Using office binary path: /usr/lib/libreoffice/program","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3817582,"logger":"api.formslibreofficeconvert.usrbinunoconv","msg":"unix process already killed","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.3819606,"logger":"api.formslibreofficeconvert.usrliblibreofficeprogramsoffice.bin","msg":"unix process killed","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"debug","ts":1652142275.384473,"logger":"api.formslibreofficeconvert","msg":"'/tmp/9fa28c36-c325-4737-8e32-421d438f1d4c' removed","trace":"6811015f-b000-49e3-9c94-2aea493be0ef"}
pdf-exporter-svc  | {"level":"error","ts":1652142275.3845446,"logger":"api","msg":"convert to PDF: unoconv PDF: unix process error: wait for unix process: exit status 5","trace":"6811015f-b000-49e3-9c94-2aea493be0ef","remote_ip":"172.18.0.1","host":"localhost:8081","uri":"/forms/libreoffice/convert","method":"POST","path":"/forms/libreoffice/convert","referer":"","user_agent":"PostmanRuntime/7.29.0","status":500,"latency":2299084627,"latency_human":"2.299084627s","bytes_in":341,"bytes_out":21}
```

**Comment by sgrigson** [comment]
Ha, Pebkac. (Using sample files stored in Dropbox after they'd been marked 'online only'.)

Using real files solved it. Derp.
